frigate/http.py

```diff
@@ -1156,27 +1157,50 @@ def latest_frame(camera_name):
     resize_quality = request.args.get("quality", default=70, type=int)
 
     if camera_name in current_app.frigate_config.cameras:
-        frame = current_app.detected_frames_processor.get_current_frame(
+        frame, frame_time = current_app.detected_frames_processor.get_current_frame(
             camera_name, draw_options
         )
         retry_interval = float(
             current_app.frigate_config.cameras.get(camera_name).ffmpeg.retry_interval
             or 10
         )
 
-        if frame is None or datetime.now().timestamp() > (
-            current_app.detected_frames_processor.get_current_frame_time(camera_name)
-            + retry_interval
-        ):
-            if current_app.camera_error_image is None:
-                error_image = glob.glob("/opt/frigate/frigate/images/camera-error.jpg")
+        latest_frame = (
+            frame_time + retry_interval * 1.5
+        )  # magic number, because of shit happens
+        now = datetime.now().timestamp()
+        if frame is None or now > latest_frame:
+            if (
```

Comment:
This is unintuitive, I think there should be a better way to know if there have been no frames received yet